### DISCUSSION TITLE
questions on Tutorial 33: Risk Parity with Constraints using the Risk Budgeting Approach

### DISCUSSION THREAD

**Discussion by xiaoguangzhu** [dcajasn/Riskfolio-Lib#171] [General]
For 3.1, what is this 0.00056488? Is it a daily return? Also why are the risk contribution not equal when a minimum target return is targeted? 

port.lowerret = 0.00056488 * 1.5

https://nbviewer.org/github/dcajasn/Riskfolio-Lib/blob/master/examples/Tutorial%2033%20-%20Risk%20Parity%20with%20Constraints%20using%20the%20Risk%20Budgeting%20Approach.ipynb 

Thanks.

**Comment by dcajasn** [comment]
Hi @xiaoguangzhu 

Yes, it is a daily return. The risk contribution per asset is different because the minimum return constraint and linear constraints adds deviation from the equal risk contribution target.

Best regards,
Dany
